x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -32,11 +32,17 @@ import { Fields, SortOption } from './types';
 import { SearchBoxView, SortingView, MultiCheckboxFacetsView } from './views';
 
 const RECENTLY_UPLOADED = i18n.translate(
-  'xpack.enterpriseSearch.appSearch.documents.search.sortBy.option.recentlyUploaded',
+  'xpack.enterpriseSearch.appSearch.documents.search.sortBy.option.documentId',
   {
-    defaultMessage: 'Recently Uploaded',
+    defaultMessage: 'Document ID',
   }
```

Comment:
This doesn't quite make sense. Why was this changed and why is the constant still called `RECENTLY_UPLOADED`?

Reply:
Got it. That makes sense. Thanks for the context!